pages/common/unzip.md

```diff
@@ -1,24 +1,25 @@
 # unzip
 
-> Extract compressed files in a ZIP archive.
+> Extract compressed files/directories from zip archives.
+> See also: `zip`.
 > More information: <https://manned.org/unzip>.
 
-- Extract zip file(s) (for multiple files, separate file paths by spaces):
+- Extract files/directories from specific archives:
 
-`unzip {{file(s)}}`
+`unzip {{path/to/archive1 path/to/archive2 ...}}`
```

Comment:
I totally get your point, that we should use more paths to show that the command can handle multiple files at once. On the other side, tough, I think this just makes the examples longer and a tiny little harder to understand. In the end we should write the examples in a way that is most useful to the user. And in 99% of the cases you want to unzip just one single archive.